src/addressResolvers/farcaster.ts

```diff
@@ -0,0 +1,107 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
+import { graphQlCall, Address, Handle, FetchError, isSilencedError, isEvmAddress } from './utils';
+
+export const NAME = 'Farcaster';
+const FNAMES_API_URL = 'https://fnames.farcaster.xyz/transfers?name=';
+const NEYNAR_API_URL = 'https://api.neynar.com/v2/farcaster/user/';
+const API_KEY = process.env.NEYNAR_API_KEY ?? '';
+
+
+interface UserDetails {
+  username: string;
+  verified_addresses: {
+    eth_addresses: string[];
+    sol_addresses: string[];
+  };
+  pfp_url: string;
+}
+
+interface ApiResponse {
+  [address: string]: UserDetails[];
+}
+
+interface UserResult {
+  username?: string;
+  eth_addresses?: string[];
+  sol_addresses?: string[];
+  pfp_url?: string;
+}
+
+
+async function fetchData<T>(url: string, method: string = 'GET'): Promise<T> {
```

Comment:
The second argument `method` does not seem to be used by any call, better to drop it